force-app/main/default/aura/CandidateGrade/CandidateGrade.cmp

```diff
@@ -4,6 +4,11 @@
     <aura:attribute name="displayGrade" type="Boolean" default="false" />
     <aura:handler name="init" value="{!this}" action="{!c.doInit}"/>
 
+    <ltng:require
```

Comment:
jQuery 1.5.1 is affected by a known vulnerability (XSS with location.hash). Please upgrade jQuery to version 1.6.3 or later.